client/web/src/enterprise/insights/pages/getting-started/components/code-insights-examples/CodeInsightsExamples.tsx

```diff
@@ -230,7 +235,7 @@ const CodeInsightCaptureExample: React.FunctionComponent<ExampleCardProps> = pro
             </div>
             <CodeInsightsQueryBlock
                 as={SyntaxHighlightedSearchQuery}
-                query="app.terraform.io/(.*)\n version =(.*)([0-9].[0-9].[0-9]) lang:Terraform archived:no fork:no"
+                query="patterntype:regexp FROMs+alpine:([d.]+) file:Dockerfile"
```

Comment:
Same correction here 
```suggestion
                query="patterntype:regexp FROM\s+alpine:([\d\.]+) file:Dockerfile"
```

Reply:
It's actually should be \\ because `\` should be used as a symbol here - a part of the query string and not as an escape operator. I check that the query with double \\ works

<img width="1289" alt="Screenshot 2022-02-17 at 12 33 19" src="https://user-images.githubusercontent.com/18492575/154447422-97f0c5fa-7fe6-4e00-a20d-2500b892cfbf.png">

I will update this PR query as well